homoalg.tex

```diff
@@ -2018,7 +2018,7 @@ \section{Komplexe und (Ko-)Homologie}
 
 \begin{defn}
   Sei $\mathcal{B}$ ein kohomologisches Koeffizientensystem auf einer simplizialen Menge $X$. Wir setzen
-  \[ C^n(X; \mathcal{B}) \coloneqq \{ \text{ Funktionen } f : (\sigma \in X_n) \to \mathcal{B}_\sigma \, \} \]
+  \[ C^n(X; \mathcal{B}) \coloneqq \{ \text{ Funktionen } f : X_n \to \coprod_{\sigma \in X_n} \mathcal{B}_\sigma | f(\sigma) \in X_n \, \} \]
```

Comment:
Cool! Again what learned. Nur wieso hat Marc das dann in der VL wieder anders notiert? Davon hab ichs nämlich übernommen.